core/src/main/java/io/kestra/core/serializers/YamlParser.java

```diff
@@ -50,7 +51,7 @@ public static  <T> T parse(Map<String, Object> input, Class<T> cls, Boolean stri
     }
 
     private static <T> String type(Class<T> cls) {
-        return cls.getSimpleName().toLowerCase();
+        return FlowInterface.class.isAssignableFrom(cls) ? "flow" : cls.getSimpleName().toLowerCase();
```

Comment:
I don't understand this change, can you explain it?

Reply:
This change is to make the error more user-friendly by returning always `flows` for any FlowInterface implementation. The error message is used both in log and in the UI. Returning `flowwithsource` as no special meaning for the user.